apps/emqx_conf/src/emqx_cluster_rpc.erl

```diff
@@ -306,6 +315,9 @@ handle_call(skip_failed_commit, _From, State = #{node := Node}) ->
 handle_call({fast_forward_to_commit, ToTnxId}, _From, State) ->
     NodeId = do_fast_forward_to_commit(ToTnxId, State),
     {reply, NodeId, State, catch_up(State)};
+handle_call(on_leave, _From, State) ->
+    {atomic, ok} = transaction(fun ?MODULE:on_leave_clean/0, []),
```

Comment:
Nit: maybe make it an anonymous fun to avoid unnecessary exports?

Reply:
I was told to be a good citizen for the hot upgrade, we should try to avoid using anonymous functions, although I know in V5 we don't support the hot upgrade now 
![bildo](https://github.com/emqx/emqx/assets/22971499/aeb33034-e59f-4c2b-a37e-33bd65b39c1c)